mono_repo.yaml

```
@@ -5,9 +5,7 @@ github:
   on:
     # Run CI on pushes to the master branch, and on PRs against master.
     push:
```

I think we want to keep this - to run the workflow on all pushes to `master` (mostly this would happen when a PR gets merged and squashed into a master commit).

```suggestion
    push:
      branches: master    
```